fl4health/clients/basic_client.py

```diff
@@ -874,6 +917,20 @@ def get_data_loaders(self, config: Config) -> Tuple[DataLoader, ...]:
         """
         raise NotImplementedError
 
+    def get_test_data_loader(self, config: Config) -> Optional[DataLoader]:
+        """
+        User defined method that returns a PyTorch Test DataLoader
+
+        Args:
+            config (Config): The config from the server.
+
+        Returns:
+            Optional[Tuple[DataLoader, ...]]: Tuple of length 2. The optinal client test loader.
+                Defaults to None.
```

Comment:
It doesn't technically "default" to none. The base implementation always returns none